codewof/static/css/third-party/bootstrap.min.css

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Bootswatch v4.3.1
+ * Homepage: https://bootswatch.com
+ * Copyright 2012-2019 Thomas Park
+ * Licensed under MIT
+ * Based on Bootstrap
```

Comment:
If it's not bootstrap should it be called bootstrap?.min.css

Reply:
I'll change it

Reply:
I just realised I need to include Bootstrap's JavaScript file.

Reply:
In the [bootswatch readme](https://github.com/thomaspark/bootswatch) it says 
>You must still include Bootstrap's JavaScript file to have functional dropdowns, modals, etc.

But when I include the required scripts (through `website.js`) it breaks the prettify css and some other css....?

I don't think this can really be approved without adding that JS file though as currently I've got hacky css that makes my dropdowns work... so if we were to add any modals etc they wouldn't work.